nvflare/fuel/f3/drivers/grpc_driver.py

```diff
@@ -262,9 +258,9 @@ def connect(self, connector: ConnectorInfo):
         self.logger.debug("CLIENT: added connection")
         try:
             received = stub.Stream(connection.generate_output())
-            connection.read_loop(received, oq)
-        except BaseException as ex:
-            self.logger.info(f"CLIENT: connection done: {type(ex)}")
+            connection.read_loop(received)
+        except Exception as ex:
+            self.logger.info(f"CLIENT: connection done: {secure_format_exception(ex)}")
         connection.close()
```

Comment:
Should we move 
"connection = StreamConnection(oq, connector, conn_props, "CLIENT", channel=channel)"
inside of the try block as in 2.3 branch?

Reply:
Yes, we should, to keep consistent between versions. Will change.

Reply:
Changed.